master/docs/developer/www.rst

```diff
@@ -617,6 +617,139 @@ Several methods are added to each "restangularized" objects, aside from get(), p
+  * call .getArray() on the returned promise to get the updating Collection before it's filled with the initial data
+
+  .. code-block:: coffeescript
+
+      # assign builds to $scope.builds once the Collection is filled
+      dataService.getBuilds(builderid: 1).then (builds) ->
+          $scope.builds = builds
+          # load steps for every build
+          builds.forEach (b) -> b.loadSteps()
+
+      # assign builds to $scope.builds before the Collection is filled using the .getArray() function
+      $scope.builds = dataService.getBuilds(builderid: 1).getArray()
+
+* ``.open(scope)``: returns a DataAccessor, handles bindings, open a new accessor every time you need updating data in a controller
+
+  .. code-block:: coffeescript
+
+      # open a new accessor every time you need updating data in a controller
+      class DemoController extends Controller
+          constructor: ($scope, dataService) ->
+              # automatically closes all the bindings when the $scope is destroyed
+              opened = dataService.open($scope)
+              # alternative syntax:
+              #   opened = dataService.open()
+              #   opened.closeOnDestroy($scope)
+              # closing it manually is also possible:
+              #   opened.close()
+
+              # request new data, it updates automatically
+              @builders = opened.getBuilders(limit: 10, order: '-started_at').getArray()
+
+* ``.control(url, method, [params])``: returns a promise, sends a JSON RPC2 POST request to the server
+
+  .. code-block:: coffeescript
+
+      # open a new accessor every time you need updating data in a controller
+      dataService.control('Forceschedulers/force', 'force').then (response) ->
+          $log.debug(response)
+      , (reason) ->
+          $log.error(reason)
+
+* ``.clearCache()``: clears the IndexedDB tables and reloads the current page
+
+  .. code-block:: coffeescript
+
+      class DemoController extends Controller
+          constructor: (@dataService) ->
+          onClick: -> @dataService.clearCache()
+
+Methods on the Wrapper classes:
+
+* ``.getXXX([id], [query])``: returns a promise<Collection>, when the promise is resolved, the Collection contains all the requested data
+
+  * same as dataService.getXXX, but with relative endpoint
+
+  .. code-block:: coffeescript
+
+      # assign builds to $scope.builds once the Collection is filled
+      dataService.getBuilds(builderid: 1).then (builds) ->
+          $scope.builds = builds
+          # get steps for every build
+          builds.forEach (b) ->
+              b.getSteps().then (steps) ->
+                  # assign completed test to every build
+                  b.complete_steps = steps.map (s) -> s.complete
+
+* ``.loadXXX([id], [query])``: returns a promise<Collection>, the Collection contains all the requested data, it is also assigned to wrapperInstance.xxx
```

Comment:
I have no idea by reading this what is the difference between loadXXX and getXXX